apps/api/.env.example

```diff
@@ -2,10 +2,6 @@
 SERVER_PORT=
 SERVER_API_KEY= # Add your API Key
```

Comment:
Even this is now removed and is fetched from database tables. Remove from development and production setup documents as well.

Reply:
Dropped this unused variable